playbook/app/pb_kits/playbook/pb_tooltip/tooltip.html.erb

```diff
@@ -7,3 +7,22 @@
     <div class="arrow" id="<%= object.tooltip_id %>-arrow"></div>
   </div>
 <% end %>
+
+<script>
+  if (<%= object.truncation_enabled %>) {
+    // only display tooltip if trigger element is truncated
+    document.addEventListener("DOMContentLoaded", function () {
+      const formatedTriggerElementId =
+        "<%= object.trigger_element_selector %>".replace("#", "");
+      const triggerElement = document.getElementById(formatedTriggerElementId);
+
+      // if scrollWidth is the same as the offsetWidth, the trigger element is not truncated
+      if (
+        triggerElement.offsetWidth === triggerElement.scrollWidth ||
+        triggerElement.scrollWidth === 0
+      ) {
+        triggerElement.classList.add("tooltip-disabled");
+      }
+    });
+  }
+</script>
```

Comment:
This will be repeated a bunch on pages using it. Any way to make this part of the main file?